hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsBlobClient.java

```
@@ -1160,8 +1160,12 @@ public AbfsRestOperation setPathProperties(final String path,
       // This path could be present as an implicit directory in FNS.
       if (op.getResult().getStatusCode() == HTTP_NOT_FOUND && isNonEmptyDirectory(path, tracingContext)) {
         // Implicit path found, create a marker blob at this path and set properties.
-        this.createPathRestOp(path, false, false, false, null,
-            contextEncryptionAdapter, tracingContext);
+        try {
+          this.createPathRestOp(path, false, false, false, null,
+              contextEncryptionAdapter, tracingContext);
+        } catch (AbfsRestOperationException exception) {
+          LOG.debug("Marker creation failed for path {} during setPathProperties", path);
```

It would be better we can log exception status code as well or some details of exception just to know what the exact reason of the failure is

---

+1 on this.
At least status code and storage error code (Enum like PATHNOTFOUND) we should print in debug log.

Also a comment here on why we are swallowing the error for future reference will be better.

---

taken